### PR TITLE
Refactor and complete the UI of all pages

### PR DESCRIPTION
**Pull request by assawalhy**

### CODE REVIEW

**Review by assawalhy — COMMENTED** [review]
LGTM

File: api/patients.ts [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+	return useMutation({
+		mutationFn: createPatient,
+		onSuccess: (response) => {
+			notifications.show({
+				title: "Patient was added successfully",
+				message: "",
+				color: "white",
+				style: { backgroundColor: "#22c55e" },
+			});
+			onSuccess?.(response);
```
Comment (mohamed-ahmed-yousef) [inline api/patients.ts:50]: what is the role of this line

File: public/file-icons/AI.svg
Comment (mohamed-ahmed-yousef) [inline public/file-icons/AI.svg:1]: which website did you use to get all of these icons?

File: global.d.ts [hunk 2]
```diff
@@ -0,0 +1,3 @@
+// Use type safe message keys with `next-intl`
+type Messages = typeof import("./messages/en.json");
+declare interface IntlMessages extends Messages {}
```
Comment (mohamed-ahmed-yousef) [inline global.d.ts:3]: Is this the only configuration to support type-safe in `next-intl`?

File: lib/cookies.server.ts [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+import { cookies } from "next/headers";
+import {
+	ACCESS_TOKEN_KEY,
+	REFRESH_TOKEN_KEY,
+	TEN_DAYS_ms,
+} from "./cookies.client";
+
+export const accessTokenCookie = createCookieStorage(ACCESS_TOKEN_KEY);
+export const refreshTokenCookie = createCookieStorage(REFRESH_TOKEN_KEY);
+
```
Comment (mohamed-ahmed-yousef) [inline lib/cookies.server.ts:12]: in which file did you set the accessTokenCookie in the server?
I see you set accessTokenCookie for the client during the login process but didn't find where you set accessTokenCookie in the server

**Review by mohamed-ahmed-yousef — APPROVED** [review]
LGTM

**Comment by mohamed-ahmed-yousef** [comment]
> LGTM

Sorry for changing your comment,  I didn't notice that

File: lib/cookies.server.ts (hunk 3 above)
Comment (assawalhy) [reply lib/cookies.server.ts:12]: Cookie doesn't live in the server. They live in the browser and sent automatically with every request to the server. Cookies are used to share settings or state between the server and the client and they are stored in the browser.

So, we can change directly the cookies in the broswer.

When you set a cookie in the server, what actually happen is that the response tells the browser to modify its cookies. So here the browser again is the storage of these cookies.

File: global.d.ts (hunk 2 above)
Comment (assawalhy) [reply global.d.ts:3]: This is what the docs say!

File: public/file-icons/AI.svg
Comment (assawalhy) [reply public/file-icons/AI.svg:1]: Googled file icons and found these neat icons. It was a Figma design, and I exported them as SVGs.

File: api/patients.ts (hunk 1 above)
Comment (assawalhy) [reply api/patients.ts:50]: Invoke the `onSuccess` function. But onSuccess may be undefined so we use the cool feature of JavaScript, optional chaining `?.`.

Comment (mohamed-ahmed-yousef) [reply api/patients.ts:50]: I think, there is no need to call it if the mutation is successful it will call automatically.

**Pull request merged**